modules/ROOT/pages/cloudhub-manage-props.adoc

```diff
@@ -92,6 +92,73 @@ Although this method is not the best practice on CloudHub (because CloudHub *Pro
 To prevent CloudHub properties from overriding properties bundled with the deployable archive, change options in the `property-placeholder` element in the Mule application.
 For more information about nondefault property placeholder options, see the https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/context/support/PropertySourcesPlaceholderConfigurer.html[Spring documentation on Property Placeholder options].
 
+== CloudHub Reserved Properties 
+
+CloudHub reserves a set of properties and prevents apps from overwriting the values of those properties.
+If an application attempts to assign a value to a reserved property, CloudHub doesn't use the assigned value.
+Instead, it returns the value as set internally without throwing an error or exception.
+
+Applications can reference reserved properties, for example, to retrieve the application name or deployment region.
+You can use the `${csorganization.id}` and `${environment.id}` properties to retrieve those values if your app makes Anypoint Platform API calls.
+You can use the `${worker.id}` to identify each worker in a multi-worker application.
+
+--
+[NOTE]
+The `${java.version}` property value doesn't include the point release for the JDK version.
+Additionally, CloudHub runtime release updates might upgrade the JDK.
+--
+
+[%header,cols="3*a"]
+|===
+|Property|Description|Example
+|`${java.vendor}`|Vendor providing the JVM|Oracle Corporation
+|`${java.runtime.name}`|JVM name|Java&#8482; SE Runtime Environment
+|`${java.version}`|JVM version|1.8.0_141
+|`${csorganization.id}`|Master or business group ID|09396371-2cd4-412e-8d08-4356dd3e7bfc
+|`${environment.id}`|Environment ID|52a69d49e4b0e3c7324ae5b6
+|`${environment.type}`|Environment type|production
+|`${worker.id}`|Worker ID (for multiworker apps)|0
+|`${domain}`|App name|`ch-https-41-demo`
+|`${fullDomain}`|Full domain name of app|`ch-https-41-demo.au-s1.cloudhub.io`
+|`${application.aws.region}`|App deployment region|`ap-southeast-2`
+|`${http.port}`|HTTP port|8081
+|`${https.port}`|HTTPS port|8082
+|`${http.private.port}`|HTTP private port|8091
+|`${https.private.port}`|HTTPS private port|8092
+|`${mule.home}`|Absolute path to Mule installation|`/opt/mule/mule-4.1.1`
+|`${spring.profiles.active}`|Used internally, *do not use*|`*****`
+|`${logging.*}`|Used internally, *do not use*|`*****`
```

Comment:
I wonder if we have to include it, or whether we should more specifically call this out as not to be used?
I worry about stuff getting lost in a table....